CounterpointGenerator/CounterpointGenerator/Generator.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace CounterpointGenerator
+{
+    public class Generator: IGenerator
+    {
+        IRuleApplier _applier;
```

Comment:
This will break the build as IRuleApplier does not yet exist.

Reply:
Now available in https://github.com/theanticrumpet/CounterpointGenerator/pull/11